src/services/request.service.js

```diff
@@ -27,4 +28,27 @@ export default class RequestService {
       }
     );
   }
+
+  /**
+   * @returns {Integer} returns number of traveled places
+   * @description This function getMostTraveledDestinations()
+   *  returns all of the destinations a and their visit times
+   */
+  static getMostTraveledDestinations = async () => {
+    const placeAndTheirVisitTimes = await sequelize.query('SELECT destination, COUNT(destination) FROM requests WHERE status !=\'pending\' AND status!=\'rejected\' GROUP BY destination ORDER BY count DESC');
```

Comment:
You will need to modify the column you're selecting

Reply:
I change it too